adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantAnnotationConverter.scala

```diff
@@ -133,19 +264,42 @@ object VariantAnnotationConverter extends Serializable {
     record
   }
 
-  private def fillKeys[T <% SpecificRecord](keys: Seq[AttrKey], vc: VariantContext, record: T): T = {
-    fillRecord(createFieldMap(keys, record.getSchema), vc, record)
-  }
-
-  def convert(vc: VariantContext, annotation: DatabaseVariantAnnotation): DatabaseVariantAnnotation = {
+  /**
+   * Remaps fields from an htsjdk variant context into a site annotation.
+   *
```

Comment:
I've been searching for a better name for `DatabaseVariantAnnotation`, could `VariantSiteAnnotation` be it?